sklearn/model_selection/_split.py

```diff
@@ -1597,3 +1597,21 @@ def _build_repr(self):
         params[key] = value
 
     return '%s(%s)' % (class_name, _pprint(params, offset=len(class_name)))
+
+
+ALL_CVS = {'KFold': KFold,
```

Comment:
`SCORERS` was public because `get_scorer` didn't exist, and because it's a register of (non-importable) strings, not a collection of classes that can be imported. I'm a little ambivalent, but this would be novel in scikit-learn. `__all__` is close to the same...?